pokemongo_bot/cell_workers/soft_ban_worker.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from pokemongo_bot.cell_workers.utils import distance
+from pokemongo_bot.worker_result import WorkerResult
+from pokemongo_bot.cell_workers import SeenFortWorker
```

Comment:
You can, but it's pretty horrible coding practice and certainly not what PEP recommends.